libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ChartConfigurationFlyout.tsx

```diff
@@ -113,13 +162,36 @@ export class ChartConfigurationFlyout extends React.Component<
       featureBasedCohortDropdownSelectedKeys.push(selectAllOptionKey);
     }
 
+    const choiceGroupOptions = [
+      {
+        key: this.datasetCohortsChoiceGroupOption,
+        text: localization.ModelAssessment.ModelOverview
+          .dataCohortsChartSelectionHeader
+      },
+      {
+        disabled: this.props.featureBasedCohorts.length === 0,
+        key: this.featureBasecCohortsChoiceGroupOption,
+        text: localization.ModelAssessment.ModelOverview
+          .featureBasedCohortsChartSelectionHeader
+      }
+    ];
+
     return (
       <Panel
         isOpen={this.props.isOpen}
         closeButtonAriaLabel="Close"
         onDismiss={this.props.onDismissFlyout}
       >
         <Stack tokens={{ childrenGap: "10px" }}>
+          <ChoiceGroup
+            options={choiceGroupOptions}
+            onChange={this.onChoiceGroupChange}
+            selectedKey={
+              this.state.datasetCohortViewIsNewlySelected
+                ? this.datasetCohortsChoiceGroupOption
+                : this.featureBasecCohortsChoiceGroupOption

```

Comment:
nit: featureBasecCohortsChoiceGroupOption -> did you mean Based instead of Basec?